azure/datalake/store/lib.py

```diff
@@ -35,15 +35,18 @@
 default_tenant = os.environ.get('azure_tenant_id', "common")
 default_username = os.environ.get('azure_username', None)
 default_password = os.environ.get('azure_password', None)
-default_client = os.environ.get('azure_client_id', "1950a258-227b-4e31-a9cf-717495945fc2")
+default_client = os.environ.get('azure_client_id', "04b07795-8ddb-461a-bbee-02f9e1bf7b46")
```

Comment:
This is the Azure CLI client ID that I am using as a default client ID for right now. Based on our previous discussion I will remove this before we publish.